### ISSUE TITLE
Some Engine.ini settings don't work.

### ISSUE THREAD

**Issue by yowave** [emoose/DQXIS-SDK#34]
I am trying to set r.DepthOfFieldQuality=0 with no success so I change it manually through the console.
Is there another place that might override my Engine.ini setting?

**Comment by mike9k1** [comment]
Yeah, probably -- see the DQ Modding Community Website here: [https://sites.google.com/view/dqxi-commands/modding/tweaks/ini-tuning-assistant](https://sites.google.com/view/dqxi-commands/modding/tweaks/ini-tuning-assistant)

**Comment by yowave** [comment]
I past it to the bottom of the Engine.ini and it still not working, i have to do it through the console.
The funny part is that when i use the console to check the status it says last seen by SystemSettingsIni.
And the value is 4, which then i change manually to 0.

I am clueless to what set it i checked my Engine.ini and there is only my r.DepthOfFieldQuality=0 and nothing else overriding it.

**Comment by user18081972** [comment]
Same problem here.
I only change the engine.ini when the game is closed.
